packages/webapp/components/history/SearchEmpty.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { AiIcon } from '@dailydotdev/shared/src/components/icons';
+import { IconSize } from '@dailydotdev/shared/src/components/Icon';
+import { SearchHistoryContainer } from './common';
+
+export function SearchEmpty(): React.ReactElement {
```

Comment:
I think we always use just `ReactElement` doesn't really matter much, but it hit my OCD 😂

Reply:
Lol. Yes, we should do it like that. Didn't notice this was not extracted.